src/test/ui/associated-types/impl-wf-cycle-1.stderr

```diff
@@ -1,14 +1,8 @@
 error[E0275]: overflow evaluating the requirement `<(T,) as Grault>::A == _`
   --> $DIR/impl-wf-cycle-1.rs:15:1
    |
-LL | / impl<T: Grault> Grault for (T,)
-LL | | where
-LL | |     Self::A: Baz,
-LL | |     Self::B: Fiz,
-...  |
-LL | |
-LL | | }
-   | |_^
```

Comment:
Interestingly, we should probably point at the first `where` clause in this error as the primary span, but that's out of scope.